BUILD.bazel

```diff
@@ -28,6 +29,9 @@ filegroup(
     ]]),
 )
 
+# TODO(alexeagle): it's messy for users to need to specify the following
```

Comment:
Can you create an issue for this? Its something that came up for me as well and it'd be nice to have a place to track the issue and discuss the best way to solve it (whether it be built into the build rule, exported by an Angular BUILD file or whatever fix we dream up)

Reply:
:+1: